azure-pipelines.yml

```
@@ -460,6 +460,12 @@ stages:
       dependsOn: runJavaBuild
       workspace:
         clean: all
+      strategy:
+        matrix:
+          CSharpITs:
```

The current version produces `Sonar.Net (Tests: Java ITs CSharpITs)` in the UI.
```suggestion
          CSharp:
```